crates/ide/src/references.rs

```diff
@@ -1270,4 +1270,27 @@ fn foo(_: bool) -> bo$0ol { true }
             "#]],
         );
     }
+
+    #[test]
+    fn test_transitive() {
+        check(
+            r#"
+//- /level3.rs new_source_root: crate:level3
+pub struct Fo$0o;
+//- /level2.rs new_source_root: crate:level2 deps:level3
+pub use level3::Foo;
+//- /level1.rs new_source_root: crate:level1 deps:level2
+pub use level2::Foo;
+//- /level0.rs new_source_root: crate:level0 deps:level1
```

Comment:
It's `new_source_root:`(note the traling `:`) as it was a simpler change this way and I don't think it's that relevant to have it be free standing?